04_numeronym/mohankrishna/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	numeronyms := make([]string, len(vals))
+	for pos, value := range vals {
+		trimmedVal := []rune(strings.Trim(value, "\n\t "))
```

Comment:
The cast to `[]rune` is unnecessary, the rest of your code works just as well without it. Removing it also allows you to removed the cast to `string` on line 24.

Reply:
The `rune` cast is added to support Unicode characters. Without that the count and charecter access is very flaky.

Reply:
Any other way of achieving this without a cast to `[]rune`?

Reply:
@anzboi - that's what I thought too at first. It is true if you only have Ascii, but for Unicode it's not true:
https://goplay.space/#R3Gni4nX9RB